src/main/java/ludum/dare/components/GamepadControlComponent.java

```diff
@@ -42,6 +43,10 @@ public GamepadControlComponent(int index) {
         }
     }
 
+    public void inControl(boolean inControl) {
```

Comment:
We can now wrestle control away from a controller/keyboard during things like when the user gets killed/hurt